gapis/service/service.proto

```diff
@@ -1329,7 +1329,22 @@ message ProfilingData {
     repeated Group groups = 3;
   }
 
+  message GpuCounters {
```

Comment:
I would get rid of this message and simply have `repeated Counter counters = 2` in the `ProfilingData` message, but I'm OK if you feel strongly about the wrapper.